### PR TITLE
3.6.1.2

### PR DESCRIPTION
**Pull request by nikhil-vr**
### This pull request addresses the following referenced issue(s):

1.3.6.1.2 related changes
2.Include a message to run step2 only after 30 mins
3.sleep for 15 mins after Storagepool creations , as there were multiple issues reported as SP taking time to initialized so its more error proof.

### Overview of changes:

1.
2.
3.

---
Notifies: @Oohdark30 @padthaitofuhot

### CODE REVIEW

File: ui/run.sh [hunk 1]
```diff
@@ -118,10 +118,11 @@ case "$(basename ${0})" in
         run ecsdeploy check || exit $?
         run ecsdeploy bootstrap || exit $?
         run ecsdeploy reboot || exit $?
-        sleep 10
+        sleep 30
```
Comment (padthaitofuhot) [inline ui/run.sh:121]: Use the correct API calls to determine cluster readiness, not sleep.

File: ui/run.sh [hunk 2]
```diff
@@ -149,6 +152,8 @@ case "$(basename ${0})" in
         o "Pinging Management API Endpoint until ready"
         run ecsconfig ping -c -x || exit $?
         run ecsconfig sp -a || exit $?
+	o "Sleeping for 15 minutes for Storage Pool to be ready"
```
Comment (padthaitofuhot) [inline ui/run.sh:155]: Use the correct API calls to determine cluster readiness, not sleep.

File: ui/run.sh [hunk 3]
```diff
@@ -141,6 +143,7 @@ case "$(basename ${0})" in
         run ping_until_clear
         run ecsdeploy deploy || exit $?
         run ecsdeploy start || exit $?
+	o "Please wait for 30 minutes before running next step for services to bootstrap"
```
Comment (padthaitofuhot) [inline ui/run.sh:146]: Use the correct API calls to determine cluster readiness, not customer behavior.

File: ui/run.sh [hunk 4]
```diff
@@ -130,6 +131,7 @@ case "$(basename ${0})" in
         run ecsdeploy bootstrap || exit $?
         run ecsdeploy deploy || exit $?
         run ecsdeploy start || exit $?
+	o "Please wait for 30 minutes before running next step for services to bootstrap"
```
Comment (padthaitofuhot) [inline ui/run.sh:134]: Use the correct API calls to determine cluster readiness, not customer behavior.

File: ui/run.sh [hunk 5]
```diff
@@ -118,10 +118,11 @@ case "$(basename ${0})" in
         run ecsdeploy check || exit $?
         run ecsdeploy bootstrap || exit $?
         run ecsdeploy reboot || exit $?
-        sleep 10
+        sleep 30
         run ping_until_clear
         run ecsdeploy deploy || exit $?
         run ecsdeploy start || exit $?
+	o "Please wait for 30 minutes before running next step for services to bootstrap"
```
Comment (padthaitofuhot) [inline ui/run.sh:125]: Use the correct API calls to determine cluster readiness, not customer behavior.

**Review by padthaitofuhot — CHANGES_REQUESTED** [review]
Please make the appropriate changes.

File: ui/run.sh (hunk 1 above)
Comment (nikhil-vr) [reply ui/run.sh:121]: This sleep was already their , it just increased to 30 seconds from 10 seconds.

File: ui/run.sh (hunk 2 above)
Comment (nikhil-vr) [reply ui/run.sh:155]: We need to check and implement API call , will take more time , something can work on 3.6.2.

File: ui/run.sh (hunk 3 above)
Comment (nikhil-vr) [reply ui/run.sh:146]: This is the end of step1 , either we should monitor the service readiness before exit or implement a service readiness check on step2.   can we move this to 3.6.2 ?

File: ui/run.sh (hunk 4 above)
Comment (nikhil-vr) [reply ui/run.sh:134]: This is the end of step1 , either we should monitor the service readiness before exit or implement a service readiness check on step2.   can we move this to 3.6.2 ?

File: ui/run.sh (hunk 5 above)
Comment (nikhil-vr) [reply ui/run.sh:125]: This is the end of step1 , either we should monitor the service readiness before exit or implement a service readiness check on step2.   can we move this to 3.6.2 ?

**Comment by nikhil-vr** [comment]
closing as we have 3.6.2 now

**Pull request closed without merging**